[Bug] CI fails to load image for testing

See #81

https://github.com/cryostatio/cryostat3/actions/runs/6509195110/job/17680185925

```
[INFO] Connected to docker: 
  Server Version: 4.6.2
  API Version: 1.41
  Operating System: ubuntu
  Total Memory: 6921 MB
Warning:  
********************************************************************************
Ryuk has been disabled. This can cause unexpected behavior in your environment.
********************************************************************************
[INFO] Checking the system...
[INFO] ✔︎ Docker server version should be at least 1.6.0
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] Starting (local) container image build for jar using docker.
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] Executing the following command to build docker image: 'docker build -f /home/runner/work/cryostat3/cryostat3/src/main/docker/Dockerfile.jvm -t quay.io/cryostat/cryostat3:3.0.0-SNAPSHOT /home/runner/work/cryostat3/cryostat3'
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #0 building with "default" instance using docker-container driver
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] 
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #1 [internal] booting buildkit
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #1 pulling image moby/buildkit:buildx-stable-1
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #1 pulling image moby/buildkit:buildx-stable-1 2.8s done
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #1 creating container buildx_buildkit_default
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #1 creating container buildx_buildkit_default 2.2s done
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #1 DONE 5.0s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] 
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #2 [internal] load build definition from Dockerfile.jvm
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #2 transferring dockerfile: 5.56kB done
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #2 DONE 0.0s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] 
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #3 [internal] load metadata for registry.access.redhat.com/ubi8/openjdk-17-runtime:1.16-2
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #3 DONE 0.5s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] 
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #4 [internal] load .dockerignore
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #4 transferring context: 143B done
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #4 DONE 0.0s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] 
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 [1/6] FROM registry.access.redhat.com/ubi8/openjdk-17-runtime:1.16-2@sha256:de58a898c3a8cbaba1732b96e3ab67994160d3ea0f649c6904e7f3dcb8fca16b
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 resolve registry.access.redhat.com/ubi8/openjdk-17-runtime:1.16-2@sha256:de58a898c3a8cbaba1732b96e3ab67994160d3ea0f649c6904e7f3dcb8fca16b done
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 sha256:5329d7039f252afc1c5d69521ef7e674f71c36b50db99b369cbb52aa9e0a6782 9.44MB / 39.33MB 0.2s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 sha256:d8b515f03629826cbe0e8d4333e6c211a736172af3d5a42f4837356bc5cccc68 0B / 91.38MB 0.2s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 sha256:5329d7039f252afc1c5d69521ef7e674f71c36b50db99b369cbb52aa9e0a6782 28.31MB / 39.33MB 0.3s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 sha256:5329d7039f252afc1c5d69521ef7e674f71c36b50db99b369cbb52aa9e0a6782 39.33MB / 39.33MB 0.4s done
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 sha256:d8b515f03629826cbe0e8d4333e6c211a736172af3d5a42f4837356bc5cccc68 11.53MB / 91.38MB 0.6s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 extracting sha256:5329d7039f252afc1c5d69521ef7e674f71c36b50db99b369cbb52aa9e0a6782
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 sha256:d8b515f03629826cbe0e8d4333e6c211a736172af3d5a42f4837356bc5cccc68 16.78MB / 91.38MB 0.8s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 sha256:d8b515f03629826cbe0e8d4333e6c211a736172af3d5a42f4837356bc5cccc68 26.21MB / 91.38MB 1.2s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 sha256:d8b515f03629826cbe0e8d4333e6c211a736172af3d5a42f4837356bc5cccc68 31.46MB / 91.38MB 1.8s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 sha256:d8b515f03629826cbe0e8d4333e6c211a736172af3d5a42f4837356bc5cccc68 36.70MB / 91.38MB 2.0s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 extracting sha256:5329d7039f252afc1c5d69521ef7e674f71c36b50db99b369cbb52aa9e0a6782 1.6s done
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 sha256:d8b515f03629826cbe0e8d4333e6c211a736172af3d5a42f4837356bc5cccc68 44.04MB / 91.38MB 2.3s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 sha256:d8b515f03629826cbe0e8d4333e6c211a736172af3d5a42f4837356bc5cccc68 52.43MB / 91.38MB 2.6s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 ...
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] 
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #6 [internal] load build context
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #6 transferring context: 75.40MB 2.9s done
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #6 DONE 2.9s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] 
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 [1/6] FROM registry.access.redhat.com/ubi8/openjdk-17-runtime:1.16-2@sha256:de58a898c3a8cbaba1732b96e3ab67994160d3ea0f649c6904e7f3dcb8fca16b
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 sha256:d8b515f03629826cbe0e8d4333e6c211a736172af3d5a42f4837356bc5cccc68 59.77MB / 91.38MB 2.9s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 sha256:d8b515f03629826cbe0e8d4333e6c211a736172af3d5a42f4837356bc5cccc68 70.25MB / 91.38MB 3.0s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 sha256:d8b515f03629826cbe0e8d4333e6c211a736172af3d5a42f4837356bc5cccc68 75.50MB / 91.38MB 3.3s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 sha256:d8b515f03629826cbe0e8d4333e6c211a736172af3d5a42f4837356bc5cccc68 88.08MB / 91.38MB 3.6s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 sha256:d8b515f03629826cbe0e8d4333e6c211a736172af3d5a42f4837356bc5cccc68 91.38MB / 91.38MB 4.1s done
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 extracting sha256:d8b515f03629826cbe0e8d4333e6c211a736172af3d5a42f4837356bc5cccc68
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 extracting sha256:d8b515f03629826cbe0e8d4333e6c211a736172af3d5a42f4837356bc5cccc68 4.2s done
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #5 DONE 8.3s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] 
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #7 [2/6] COPY --chown=185 src/main/docker/include/cryostat.jfc /usr/lib/jvm/jre/lib/jfr/
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #7 DONE 0.0s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] 
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #8 [3/6] COPY --chown=185 target/quarkus-app/lib/ /deployments/lib/
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #8 DONE 0.3s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] 
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #9 [4/6] COPY --chown=185 target/quarkus-app/*.jar /deployments/
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #9 DONE 0.0s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] 
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #10 [5/6] COPY --chown=185 target/quarkus-app/app/ /deployments/app/
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #10 DONE 0.0s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] 
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #11 [6/6] COPY --chown=185 target/quarkus-app/quarkus/ /deployments/quarkus/
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] #11 DONE 0.0s
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] WARNING: No output specified with docker-container driver. Build result will only remain in the build cache. To push result image into registry use --push or to load image into docker use --load
[INFO] [io.quarkus.container.image.docker.deployment.DockerProcessor] Built container image quay.io/cryostat/cryostat3:3.0.0-SNAPSHOT (null)

Error response from daemon: failed to find image quay.io/cryostat/cryostat3:3.0.0-SNAPSHOT: quay.io/cryostat/cryostat3:3.0.0-SNAPSHOT: image not known
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  05:27 min
[INFO] Finished at: 2023-10-13T14:08:25Z
[INFO] ------------------------------------------------------------------------
Error:  Failed to execute goal io.quarkus.platform:quarkus-maven-plugin:3.2.4.Final:build (build) on project cryostat3: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
Error: [ERROR] 	[error]: Build step io.quarkus.container.image.docker.deployment.DockerProcessor#dockerBuildFromJar threw an exception: java.lang.RuntimeException: Execution of 'docker tag quay.io/cryostat/cryostat3:3.0.0-SNAPSHOT quay.io/cryostat/cryostat3:dev' failed. See docker output for more details
Error:  	at io.quarkus.container.image.docker.deployment.DockerProcessor.dockerException(DockerProcessor.java:349)
Error:  	at io.quarkus.container.image.docker.deployment.DockerProcessor.createAdditionalTags(DockerProcessor.java:332)
Error:  	at io.quarkus.container.image.docker.deployment.DockerProcessor.createContainerImage(DockerProcessor.java:223)
Error:  	at io.quarkus.container.image.docker.deployment.DockerProcessor.dockerBuildFromJar(DockerProcessor.java:110)
Error:  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
Error:  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
Error:  	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
Error:  	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
Error:  	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:864)
Error:  	at io.quarkus.builder.BuildContext.run(BuildContext.java:282)
Error:  	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
Error:  	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2513)
Error:  	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1538)
Error:  	at java.base/java.lang.Thread.run(Thread.java:833)
Error:  	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
Error:  -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[INFO] Recoverable I/O exception (java.io.IOException) caught when processing request to {}->unix://localhost:2375
Error: Process completed with exit code 1.
```

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -88,8 +88,7 @@ jobs:
         mkdir -p $HOME/.bin
         cat <(echo '#!/usr/bin/env bash') <(echo 'exec podman "$@"') > $HOME/.bin/docker
         chmod +x $HOME/.bin/docker
-        export PATH="$HOME/.bin:$PATH"
-        docker --version
+        echo "PATH=$HOME/.bin:$PATH" >> "$GITHUB_ENV"
     - name: Set up testcontainers for podman
       run:  echo ryuk.container.privileged=true > ~/.testcontainers.properties
     - name: Start Podman API

